typescript/introduction/tsc-interfaces-classes/tsc-classes-intro.md

```diff
@@ -0,0 +1,77 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Classes Intro
+
+---
+## Content
+
+Classes represent the building blocks of the style of programming called *object-oriented (class-based) programming*, or OOP[1]. 
+
+The idea behind it is to organize related values and behaviors into definitions of entities that exist in your program (an "Animal", a "GamePlayer", a "User", etc). Then you can use these definitions, called classes, to *create (or instantiate)* those entities wherever you'd like. These instances of classes are called objects.
```

Comment:
Maybe add all 4 OO concepts - Abstraction, Encapsulation, Inheritance, Polymorphism?

Reply:
Hmmm maybe in the footnote (it's not beginner-friendly at this point)